ntcore/src/main/java/edu/wpi/first/networktables/NetworkTableType.java

```diff
@@ -48,10 +51,84 @@ public static NetworkTableType getFromInt(int value) {
         return kDoubleArray;
       case 0x40:
         return kStringArray;
-      case 0x80:
-        return kRpc;
+      case 0x100:
+        return kInteger;
+      case 0x200:
+        return kFloat;
+      case 0x400:
+        return kIntegerArray;
+      case 0x800:
+        return kFloatArray;
       default:
         return kUnassigned;
     }
   }
+
+  /**
+   * Convert from a type string to an enum type.
+   *
+   * @param typeString type string
+   * @return The kind
+   */
+  public static NetworkTableType getFromString(String typeString) {
+    if ("boolean".equals(typeString)) {
```

Comment:
Replace with switch?
Also, perhaps have these strings as constants somewhere?

Reply:
Ah, I didn't realize string switch was a thing in Java.  I'll make that change.